src/Core/Tools/Enums/ReferenceEventType.cs

```diff
@@ -42,6 +42,8 @@ public enum ReferenceEventType
     OrganizationEditedByAdmin,
     [EnumMember(Value = "organization-created-by-admin")]
     OrganizationCreatedByAdmin,
+    [EnumMember(Value = "organization-edited-in-stripe")]
+    OrganizationEditedInStripe,
```

Comment:
I don't see this used anywhere? Is this part of some other work?

Reply:
Dicussed this with @amorask-bitwarden and I just missed the usage in the StripeController. Changes are looking good.